apps/frontend/src/components/cards/controltable/ControlRowHeader.vue

```diff
@@ -71,6 +71,11 @@
     <template #id>
       <v-card-text class="pa-2 title font-weight-bold">
         {{ control.data.id }}
+        {{
```

Comment:
This only puts things on a new line by a fluke on non-widescreen (1440p and up) resolutions.  My suggestion is to put the control.data.id in a div and then the legacy tag stuff in a separate div.  Make that separate div have a v-if that does the checking for if the legacy tag exists.

Reply:
As long as this means the (legacy ID) will be on a new line underneath thank you!